libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Settings/RuntimeSettings.cs

```diff
@@ -37,12 +36,12 @@ internal class RuntimeSettings
         public SkillSettings Skills { get; set; } = new SkillSettings();
 
         /// <summary>
-        /// Gets or sets the list of plugins registered for the runtime.
+        /// Gets or sets the list of components registered for the runtime.
         /// </summary>
         /// <value>
-        /// The list of plugins registered for the runtime.
+        /// The list of components registered for the runtime.
         /// </value>
-        public IList<BotPluginDefinition> Plugins { get; set; } = new List<BotPluginDefinition>();
+        public IList<BotComponentDefinition> Plugins { get; set; } = new List<BotComponentDefinition>();
```

Comment:
No telling about how much this would effect down the line, but why wouldn't "plugins" in config be changed to "components"?

Reply:
Tracy and I talked offline. Yes, this is coming in a few mins.

Reply:
Done